src/Mvc/Mvc.Razor/src/Infrastructure/DefaultFileVersionProvider.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using Microsoft.AspNetCore.Antiforgery;
```

Comment:
@JunTaoLuo is this still needed?

Reply:
Nope. Should have removed it.

Reply:
Unresolving 'til I make the change here.